src/modelbench/static_site_generator.py

```diff
@@ -121,7 +121,12 @@ def content_string(self, item: str, key: str):
 
     @content.register
     def content_sut(self, item: SutDescription, key: str):
-        return self._content[item.key][key]
+        if item.key in self._content:
+            return self._content[item.key][key]
+        elif key == "name":
+            return item.display_name
```

Comment:
Since this is all we care about, should we remove the `tagline` from the .toml file now?

Reply:
Removed!